controllers/front/payment.php

```diff
@@ -127,7 +130,8 @@ public function postProcess()
                 ->setCountryCode('ES')
                 ->setCity($billingAddress->city)
                 ->setAddress($billingAddress->address1 . ' ' . $billingAddress->address2)
-                ->setDni($billingAddress->dni)
```

Comment:
Taking a look to last version, the code contains the line: ->setDni($billingAddress->dni). I have been trying to find the variable DNI in the methods getNationalId and I did not find it.

Reply:
Added dni variable